stats/stats.py

```diff
@@ -74,23 +74,29 @@ class Stats(object):
             }
         },
         'subscriptions': {
+            'estimated_income':{
                 'modifiers': ['include_unused_plans'],
                 'granulations': {
-                    'plan': None,
-                    'month': ['Jan']
+                    'billing_date': ['year', 'month'],
```

Comment:
`year` and `month` are periods or intervals. Maybe rename `billing_date` to `billing_period` or something simillar.